presto-main/src/main/java/io/prestosql/sql/planner/PlanOptimizers.java

```diff
@@ -45,7 +45,9 @@
 import io.prestosql.sql.planner.iterative.rule.ExtractSpatialJoins;
 import io.prestosql.sql.planner.iterative.rule.GatherAndMergeWindows;
 import io.prestosql.sql.planner.iterative.rule.ImplementBernoulliSampleAsFilter;
+import io.prestosql.sql.planner.iterative.rule.ImplementExceptAsUnion;
```

Comment:
What about the existing `ImplementIntersectAndExceptAsUnion`? Did you leave it intentionally?